### DISCUSSION TITLE
Translange some games to the portuguese.

### DISCUSSION THREAD

**Discussion by Solkrafty** [nesbox/TIC-80#2436] [Ideas]
Hi!
I really want to help you'res grow this community, and I thought about doing it by translating games! Do you'res think it's a good idea?

**Comment by EricLSSR** [comment]
Brazilian bro ?

Posso comentar em português man ?

Séria uma boa ideia mas seria necessário da permissão dos criadores dos jogos. Recomendo tu ir atrás do criador do jogo pra perguntar isso diretamente a ele. Pra ver se não gera um incômodo nele

**Comment by bettywolfy** [comment]
Uma tradução pro Tic80 não parece necessária haha
